hs_geographic_feature_resource/receivers.py

```diff
@@ -312,50 +316,55 @@ def validate_form(request, element_name):
 def geofeature_pre_delete_file_from_resource(sender, **kwargs):
 
     res_obj = kwargs['resource']
-    del_file = kwargs['file']
-    one_file_removed = True
-    all_file_removed = False
-    del_res_fname = get_resource_file_name(del_file)
+    res_id = res_obj.short_id
+    try:
```

Comment:
just moved existing code into a "try...except" statement. git traced all changed lines because of new indent.